### PR TITLE
DropDownList and Lookup: get rid of qunit-fixture styling

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js [hunk 1]
```diff
@@ -1146,11 +1146,17 @@ QUnit.module('popup', moduleConfig, () => {
...
-            .css('left', 0)
-            .css('top', 0);
+        const parentContainer = $('<div>')
+            .css({
+                left: 0,
+                top: 0
+            })
+            .appendTo('#qunit-fixture');
+        const $dropDownList = $('<div>')
+            .attr('id', 'innerDropDownList')
```
Comment (alexander-kotov-dx) [inline testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: Are we use this id anywhere?

Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: fixed

**Review by alexander-kotov-dx — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js [hunk 2]
```diff
@@ -1146,11 +1146,15 @@ QUnit.module('popup', moduleConfig, () => {
...
+        const parentContainer = $('<div>')
+            .css({
+                left: 0,
+                top: 0
+            })
+            .appendTo('#qunit-fixture');
+        const $dropDownList = $('<div>').appendTo(parentContainer);
 
-        $('#dropDownList').dxDropDownList({
+        $dropDownList.dxDropDownList({
```
Comment (San4es) [inline testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]:
```
$('#dropDownList')
    .wrap($('<div>).css({ ... }))
     .dxDropDownList({
    ...
```
?

**Review by DokaRus — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js (hunk 2 above)
Comment (ksercs) [reply testing/tests/DevExpress.ui.widgets.editors/dropDownList.tests.js]: fixed

**Review by San4es — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**